src/redux/actions/organisationUnits.js

```diff
@@ -7,23 +7,24 @@ export const ACTION_TYPES = {
     ORGANISATION_UNIT_SELECTED: 'ORGANISATION_UNIT_SELECTED',
 }
 
+export const errorMessage = i18n.t('Could not load organisation units')
+
 export const loadOrganisationUnitsSuccess = periodTypes => ({
     type: ACTION_TYPES.ORGANISATION_UNITS_RECEIVED,
     payload: periodTypes,
 })
 
 export const loadOrganisationUnitsError = () => ({
     type: ACTION_TYPES.ORGANISATION_UNITS_ERRORED,
-    payload: i18n.t('Could not load organisation units'),
+    payload: errorMessage,
```

Comment:
I think it makes sense to get the errorMessage from a func parameter and set the default message as the default value of the parameter. That way we could also provide more meaningful error messages if the api returns error messages like 'you don't have the permissions to ....'